chain/index/read.go

```diff
@@ -49,6 +50,9 @@ func (si *SqliteIndexer) GetMsgInfo(ctx context.Context, messageCid cid.Cid) (*M
 	var height int64
 
 	if err := si.queryMsgInfo(ctx, messageCid, &tipsetKeyCidBytes, &height); err != nil {
+		if strings.Contains(err.Error(), "no rows in result set") {
```

Comment:
I think `if err == sql.ErrNoRows {` should have been OK for this, no? You had it in your original version, was it not working? `errors.Is(err, sql.ErrNoRows)` if we're afraid about it being wrapped.

Reply:
It wasn't working as intended while testing out `require.Eventually` test case, which is why I changed it, I'll revert it and if everything works as needed I'll push the changes